pybamm/models/submodels/interface/open_circuit_potential/plett_ocp.py

```diff
@@ -0,0 +1,151 @@
+#
+# from Wycisk 2022
+#
+import pybamm
+from . import BaseOpenCircuitPotential
+
+
+class PlettOpenCircuitPotential(BaseOpenCircuitPotential):
+    def get_fundamental_variables(self):
+        domain, Domain = self.domain_Domain
+        phase_name = self.phase_name
+        h = pybamm.Variable(
+            f"{Domain} electrode {phase_name}hysteresis state",
+            domains={
+                "primary": f"{domain} electrode",
+                "secondary": "current collector",
+            },
+        )
+        return {
+            f"{Domain} electrode {phase_name}hysteresis state": h,
+        }
+
+    def get_coupled_variables(self, variables):
+        domain, Domain = self.domain_Domain
+        phase_name = self.phase_name
+        phase = self.phase
+
+        # get dQ, change in capacity with respect to stoichiometry
+        c_max = self.phase_param.c_max
+        epsilon_s_av = self.phase_param.epsilon_s_av
+        V_electrode = self.param.A_cc * self.domain_param.L
+        Li_max = c_max * V_electrode * epsilon_s_av
+        Q_max = Li_max * self.param.F / 3600
+        dQ = Q_max
```

Comment:
This isn't `dQ`, right, it's just the electrode capacity? I think later on you are writing `dU/dQ = dU/dx * dx/dQ`? Can you point to which equation the paper you've implemented as they seem to express it a few ways in the paper?

Reply:
Yeah my thinking here is that dQ(sto) would be equal to Q*sto, so maybe I need to multiply it by sto.

```suggestion
        dQ = Q_max * sto_bulk
```

Reply:
I believe it wasn't dQ. I believe I have since fixed it, and reverted back to the functional form of Q and taking the differential of that. 

Here is the equation:
<img width="648" alt="image" src="https://github.com/pybamm-team/PyBaMM/assets/140573653/7ecd3ab8-eca3-40b7-93f2-5e870b97ddb8">